scripts/post-upgrade.sh

```diff
@@ -0,0 +1,46 @@
+#! /bin/bash
+set -ex
+
+if [ "$#" -ne 6 ]; then
+	echo "Error : Invalid number of arguments"
+	Usage: "./post-upgrade.sh <pravegacluster name> <pravega-release-name> <bookkeeper-release-name> <version> <namespace> <zk-svc-name>"
+	exit 1
+fi
+
+name=$1
+pname=$2
+bkname=$3
+version=$4
+namespace=$5
+zksvc=$6
+
+kubectl describe PravegaCluster $name -n $namespace
+currentReplicasPc=`kubectl get PravegaCluster $name -n $namespace -o jsonpath='{.status.currentReplicas}'`
+readyReplicasPc=`kubectl get PravegaCluster $name -n $namespace -o jsonpath='{.status.readyReplicas}'`
+if [ $currentReplicasPc != $readyReplicasPc ]; then
+  exit 1
```

Comment:
When exiting with non-zero, it could be useful to warn the user of _why_ (in this case, not all the replicas are ready), and _how_ they can fix it / what they need to do to proceed

Reply:
Added the reason for failure in `echo` commands